docs/howto/krustlet-on-gke.md

```diff
@@ -0,0 +1,294 @@
+
+```shell
+$ cat <<EOF | kubectl apply --filename=-
+apiVersion: certificates.k8s.io/v1beta1
+kind: CertificateSigningRequest
+metadata:
+  name: krustlet
+spec:
+  request: $(cat krustlet.csr | base64 | tr -d '\n')
+  usages:
+  - digital signature
+  - key encipherment
+  - server auth
+EOF
+certificatesigningrequest.certificates.k8s.io/krustlet created
+```
+
+You should then approve the request:
+
+```shell
+$ kubectl certificate approve krustlet
+certificatesigningrequest.certificates.k8s.io/krustlet approved
+```
+
+After approval, you can download the cert like so:
+
+```shell
+$ kubectl get csr krustlet --output=jsonpath='{.status.certificate}' \
+    | base64 --decode > krustlet.crt
+```
+
+Lastly, combine the key and the cert into a PFX bundle, choosing your own password instead of
+"password":
+
+```shell
+$ openssl pkcs12 -export -out krustlet.pfx -inkey krustlet.key -in krustlet.crt -password "pass:password"
+```
+
+## Step 3: Copy assets to VM
+
+The first thing we'll need to do is copy up the assets we generated in steps 1 and 2. Copy them to the VM by typing:
+
+```shell
+$ gcloud compute scp krustlet.pfx ${INSTANCE}: --project=${PROJECT} --zone=${ZONE}
+$ gcloud compute scp kubeconfig-sa ${INSTANCE}: --project=${PROJECT} --zone=${ZONE}
+```
+
+We can then SSH into the instance by typing:
+
+```shell
+$ gcloud compute ssh ${INSTANCE} --project=${PROJECT} --zone=${ZONE}
+```
+
+## Step 4: Install and configure Krustlet
+
+Install the latest release of krustlet following [the install guide](../intro/install.md).
+
+There are two flavors of Krustlet (`krustlet-wasi` and `krustlet-wascc`), let's use the first:
+
+```shell
+$ KUBECONFIG=${PWD}/kubeconfig-sa ./krustlet-wasi \
+--node-ip=${IP} \
+--node-name="krustlet" \
+--pfx-password="password" \
+--pfx-path=./krustlet.pfx
+```
+
+> **NOTE** To increase the level of debugging, you may prefix the command with `RUST_LOG=info` or `RUST_LOG=debug`.
+
+> **NOTE** The value of `${IP}` was determined in step #1.
+
+
+
+From another terminal that's configured to access the cluster, you should be able to enumerate the cluster's nodes including the Krustlet by typing:
+
+```shell
+$ kubectl get nodes
+NAME                                     STATUS   ROLES    AGE   VERSION
+gke-cluster-default-pool-1a3a5b85-scds   Ready    <none>   59m   v1.17.4-gke.10
+gke-cluster-default-pool-3885c0e3-6zw2   Ready    <none>   36m   v1.17.4-gke.10
+gke-cluster-default-pool-6d70a85d-19r8   Ready    <none>   59m   v1.17.4-gke.10
+krustlet                                 Ready    agent    8s    v1.17.0
+```
+
+## Step 5: Test that things work
+
+We may test that the Krustlet is working by running one of the demos:
+
+```shell
+$ kubectl apply --filename=https://raw.githubusercontent.com/deislabs/krustlet/master/demos/wasi/hello-world-rust/k8s.yaml
+$ wait a few seconds for the pod to run
```

Comment:
This should be a comment `#` and not `$`

Reply:
Apologies, yes. This syntax continues to unfamiliar to me.

Fixed: 66f22d5d20b79f286aa3470292b1bbef48281062